cognite/client/_api/assets.py

```diff
@@ -48,6 +50,15 @@ def __call__(
             Union[Asset, AssetList]: yields Asset one by one if chunk is not specified, else AssetList objects.
         """
 
+        if root_ids and isinstance(root_ids[0], dict):
+            if root_external_ids:
+                raise ValueError(
```

Comment:
Feel like this validation is redundant if we don't document the use of objects for `root_ids`.

Reply:
fair enough